iocage/iocage.8

```diff
@@ -774,8 +979,9 @@ interface.
 A netmask in either dotted-quad or CIDR form given after the IP
 address is used when adding the IP alias.
 .Pp
-The IP address is automatically assigned at the first start of the jail.
-This requires that the
+The IP address is automatically assigned at the first start of the
+jail.
+This requires the
 .Dv ip4_autostart
```

Comment:
`autostart` and `autoend` don't exist yet. They may or may not return in the same form, so we should nuke references to them.